includes/documents/abstract-wcpdf-order-document-methods.php

```diff
@@ -1207,6 +1207,18 @@ public function invoice_date() {
 		echo $this->get_invoice_date();
 	}
 
+	public function get_document_notes( $template_type ) {
```

Comment:
do we need the `$template_type` argument here? I think we can safely assume that 99.99% of usage will be within the same (current) document. For the exceptions (custom setups) people can load that document by themselves I think?